src/System.Runtime/tests/System/GC.cs

```diff
@@ -357,4 +358,42 @@ public static void TestGetGeneration()
             GC.Collect();
         }
     }
+
+    [Theory]
+    [InlineData(GCLargeObjectHeapCompactionMode.CompactOnce)]
+    [InlineData(GCLargeObjectHeapCompactionMode.Default)]
+    public static void TestLargeObjectHeapCompactionModeRoundTrips(GCLargeObjectHeapCompactionMode value)
+    {
+        GCLargeObjectHeapCompactionMode orig = GCSettings.LargeObjectHeapCompactionMode;
+        try
+        {
+            GCSettings.LargeObjectHeapCompactionMode = value;
+            Assert.Equal(value, GCSettings.LargeObjectHeapCompactionMode);
+        }
+        finally
+        {
+            GCSettings.LargeObjectHeapCompactionMode = orig;
+            Assert.Equal(orig, GCSettings.LargeObjectHeapCompactionMode);
+        }
+    }
+
+    [Theory]
+    [InlineData(GCLatencyMode.Batch)]
+    [InlineData(GCLatencyMode.Interactive)]
+    //[InlineData(GCLatencyMode.LowLatency)] Concurent GC is not enabled on Unix
+    //[InlineData(GCLatencyMode.SustainedLowLatency)] 
```

Comment:
Rather than just commenting them out, seems like it'd be better to either a) remove them if we don't think these matter, or b) add a test like:
```C#
[ActiveIssue(..., PlatformID.AnyUnix)] // recombine into TestLatencyRoundtrips once addressed
[Theory]
[InlineData(GCLatencyMode.LowLatency)]
[InlineData(GCLatencyMode.SustainedLowLatency)]
public static void TestLatencyRoundtrips_LowLatency(GCLatencyMode value) => TestLatencyRoundtrips(value);
```